tests/base.py

```diff
@@ -19,6 +19,12 @@
 
 class BaseTest(unittest.TestCase):
 
+  def assertStrictTrue(self, to_assert):
+    self.assertIs(to_assert, True)
+
+  def assertStrictFalse(self, to_assert):
+    self.assertIs(to_assert, False)
+
```

Comment:
assertTrue and assertFalse check for truthy and falsy values. 
Added these to ensure that we don't mistake null for false